woocommerce/woocommerce-android
# Activate: Add notification permission request to store creation

Fixes #9061, a subtask of #8999.

[device-2023-05-19-170735.webm](https://github.com/woocommerce/woocommerce-android/assets/1522856/fa2080e1-8465-4901-992a-27c42274e5a1)

**To test:**
1. Run the app on a device running API 33
2. Start the store creation flow
3. Create a new account
4. When you enter the store name picker, notice a notification permission request is displayed
5. Tap on Allow
6. Go to the app settings and verify the Notifications permission is listed in the allowed permissions
7. Alternatively, if you deny the permission and re-enter the name picker, a dialog with a permission rationale is shown
